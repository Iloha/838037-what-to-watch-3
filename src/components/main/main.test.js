import React from "react";
import renderer from "react-test-renderer";
import Main from "./main.jsx";

const films = [
  {
    name: `Fantastic Beasts: The Crimes of Grindelwald`,
    genre: `comedy`,
    preview: `fantastic-beasts-the-crimes-of-grindelwald.jpg`
  }
];

it(`<Main /> should render MovieList component`, () => {
  const tree = renderer
    .create(<Main
      films={films}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
