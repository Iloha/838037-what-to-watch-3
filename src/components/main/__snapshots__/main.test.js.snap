// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render movie names 1`] = `
Array [
  <div>
    FFantastic Beasts
  </div>,
  <div>
    BBohemian Rhapsody
  </div>,
  <div>
    MMacbeth
  </div>,
]
`;
